src/Syncromatics.Clients.Metro.Api/IMetroApi.cs

```diff
@@ -1,22 +1,30 @@
+using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using RestEase;
 using Syncromatics.Clients.Metro.Api.Models;
 
 namespace Syncromatics.Clients.Metro.Api
 {
+    [Header("User-Agent", "Syncromatics.Clients.Metro")]
     public interface IMetroApi
     {
-        [Get("/api/node_time.php")]
-        Task<NodeTimesResponse> GetNodeTimes(string node_id, string format = "json");
-        
-        [Get("/api/node_time.php")]
-        Task<NodeTimesResponse> GetStopTimes(string stop_id, string format = "json");
-        
-        [Get("/API/=stops_SYNC/Stops_N_Node.php")]
-        Task<List<Stop>> GetStops(string query_type, string node_IDWC = null, string node = null, string stop_id = null, string carrier_code = null, string minifyresult = "true", string output_format = "json");
+        [Get("/api_test/node_time.php?format=json")]
+        Task<NodeTimesResponse> GetNodeTimes(string node_id, string getonoff = "on");
```

Comment:
Can you use StopPassageOption instead of string for getonoff?

Reply:
I ran down this road initially. RestEase allows you to define a query serializer for the client, so I tried to do that, but found that I had to implement quite a bit more than I wanted to in order to handle this one case. So I did the enum handling in the MetroApiClient instead.

Reply:
You can do this relatively easy by inheriting from Dictionary<string,string> on your object and using QueryMapAttribute